docs/docs/cli-reference/cluster-get.md

```diff
@@ -0,0 +1,34 @@
+---
+title: cluster get
+sidebar_label: cluster get
+slug: /cluster-get
+---
+
+Kurtosis will work locally or over remote infrastructure. To determine the type of infrastructure your instance of Kurtosis is running on, simply run:
+
+```bash
+kurtosis cluster get
+```
+
+The clusters that Kurtosis can connect to are defined in your `kurtosis-config.yml` file, located at `/Users/YOUR_USER/Library/Application Support/kurtosis/kurtosis-config.yml` on MacOS. See [this guide](https://docs.kurtosis.com/k8s#iii-add-your-cluster-information-to-kurtosis-configyml) to learn more about how to add cluster information to your `kurtosis-config.yml` file.
```

Comment:
Suggestion: encapsulate YOUR_USER in brackets like <YOUR_USER> just to make it more obvious the reader has to switch it out

Reply:
Roger, I copied it from https://docs.kurtosis.com/next/k8s#iii-add-your-cluster-information-to-kurtosis-configyml but can change it now